packages/core/build/config.js

```diff
@@ -18,7 +18,7 @@ const copyConfig = base => {
             },
             {
                 from: path.resolve(__dirname, '../node_modules/@deriv/dashboard/lib/assets/images'),
-                to: 'js/dashboard/assets/images',
+                to: 'public/images',
```

Comment:
we need to specify the dashboard assets path, can we revert this 🙏

Reply:
@reza-fs  Could you please take a look at this comment?